setup/www/tools/promote/_resha.sh

```diff
@@ -16,7 +16,10 @@ if [ "X${version}" == "X" ]; then
   exit 1
 fi
 
-(cd "${dstdir}/${version}" && shasum -a256 $(ls node* iojs* win-*/* 2> /dev/null) > SHASUMS256.txt) || exit 1
+(cd "${dstdir}/${version}" && shasum -a256 $(ls node* openssl* iojs* win-*/* x64/* 2> /dev/null) > SHASUMS256.txt) || exit 1
```

Comment:
<=v0.12 keeps win-32 in ./ and win-64 in x64/, they have additional node.* files and add openssl* files, they also add a SHA-1 SHASUMS.txt